In [1]:
import pandas as pd

Tener que hacer los Slide , limpiar nulos faltantes, corregir errores topograficos de cocina, cambiar Cp a object y rellenar nulos,rellenar nulos calle_cruce

In [14]:
df = pd.read_csv('oferta_gastronomica_limpio.csv', sep=';', encoding='latin-1', on_bad_lines='skip')

In [4]:
df

,long,lat,id,nombre,categoria,cocina,ambientacion,telefono,mail,horario,calle_nombre,calle_altura,calle_cruce,direccion_completa,barrio,comuna,codigo_postal,codigo_postal_argentino
0,-58.363307,-34.622963,1602,GUANTANAMERA,RESTAURANTE,CUBANA,MODERNO,4314 0770,NaN,NO ESPECIFICADA,MOREAU DE JUSTO ALICIA,2092,NaN,"MOREAU DE JUSTO, ALICIA AV. 2092",Puerto Madero,1,1107.0,C1107AFP
1,-58.392330,-34.621554,676,1843 VINACOTECA,VINERIA,GASTRONOMIA GENERAL,GENERAL,4779 0806,NaN,lunes a domingo de 8 a 00,HUMBERTO PRIMO,1843,NaN,HUMBERTO 1� 1843,San Cristobal,3,NaN,CnanXXX
2,-58.453542,-34.560311,487,2080 EXPRESS,RESTAURANTE,TRADICIONAL,CLASICO PORTENIO,4788 4500,NaN,lunes a domingo de 8 a 00,JURAMENTO,2080,NaN,JURAMENTO AV. 2080,Belgrano,13,1428.0,C1428DNH
3,-58.475491,-34.677051,2737,22,CAFE,MINUTAS,MODERNO,NO DISPONIBLE,NaN,NO ESPECIFICADA,MURGUIONDO,4188,NaN,MURGUIONDO 4188,Villa Lugano,8,1439.0,C1439FTB
4,-58.425437,-34.585695,2,4 EN 2 I,RESTAURANTE,GASTRONOMIA GENERAL,GENERAL,4832 9510,NaN,NO ESPECIFICADA,BORGES JORGE LUIS,2095,NaN,"BORGES, JORGE LUIS 2095",Palermo,14,1425.0,C1425FFA
...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...
2818,-58.401107,-34.599643,893,ZOE PIZZA & CAFE,RESTAURANTE,PORTEÑA,MODERNO,4961 8378,NaN,NO ESPECIFICADA,AZCUENAGA,796,NaN,AZCUENAGA 796,Balvanera,3,1029.0,C1029AAP
2819,-58.386372,-34.607178,2546,ZOMBI,RESTAURANTE,MINUTAS,MODERNO,4371 6964,NaN,lunes a viernes de 8 a 00,URUGUAY,142,NaN,URUGUAY 142,San Nicolas,1,1015.0,C1015ABD
2820,-58.463108,-34.543369,1521,ZONA,RESTAURANTE,INTERNACIONAL,RUSTICA,4704 5018,www.zonapizza.com,NO ESPECIFICADA,11 DE SEPTIEMBRE DE 1888,3900,NaN,11 DE SEPTIEMBRE DE 1888 3900,Nunez,13,1429.0,C1429BIV
2821,-58.395697,-34.602213,894,ZOO BAR,RESTAURANTE,MINUTAS,MODERNO,4374 4879,NaN,NO ESPECIFICADA,TUCUMAN,2002,NaN,TUCUMAN 2002,Balvanera,3,1050.0,C1050AAP


In [17]:
df = pd.read_csv('oferta_gastronomica_limpio.csv', sep=';', encoding='latin-1', on_bad_lines='skip')

df.info()

<class 'pandas.DataFrame'>
RangeIndex: 2823 entries, 0 to 2822
Data columns (total 18 columns):
 #   Column                   Non-Null Count  Dtype  
---  ------                   --------------  -----  
 0   long                     2823 non-null   float64
 1   lat                      2823 non-null   float64
 2   id                       2823 non-null   int64  
 3   nombre                   2823 non-null   str    
 4   categoria                2823 non-null   str    
 5   cocina                   2823 non-null   str    
 6   ambientacion             2823 non-null   str    
 7   telefono                 2823 non-null   str    
 8   mail                     472 non-null    str    
 9   horario                  2823 non-null   str    
 10  calle_nombre             2823 non-null   str    
 11  calle_altura             2823 non-null   int64  
 12  calle_cruce              6 non-null      str    
 13  direccion_completa       2823 non-null   str    
 14  barrio                   2823 non-n

In [53]:
df = pd.read_csv('oferta_gastronomica_limpio.csv', sep=';', encoding='latin-1', on_bad_lines='skip')

df["cocina"].value_counts()

cocina
MINUTAS                737
INTERNACIONAL          448
GASTRONOMIA GENERAL    359
TRADICIONAL            297
PIZZERIA               176
PARRILLA               159
SANDWICHERIA           127
FAST FOOD               81
ITALIANA                77
MEDITERRANEA            48
PORTEÑA                39
DE AUTOR                32
REGIONAL                30
ESPAÑOLA               25
CHINA                   17
SUSHI                   12
TENEDOR LIBRE           11
JAPONESA                10
AMERICANA                9
FUSION                   8
FRANCESA                 8
TAPAS                    8
NATURISTA                7
MEXICANA                 7
MARISCOS                 7
MODERNO                  7
CUBANA                   6
ARABE                    6
ARMENIA                  6
BOLIVIANA                6
GOURMET                  5
CLASICO PORTEÑO         4
ORIENTAL                 4
VEGETARIANA              4
PERUANA                  4
ALEMANA                  3
PATAGONICA           

In [39]:
df["ambientacion"].value_counts()

ambientacion
CLASICO PORTENIO        898
MODERNO                 875
GENERAL                 559
ANTIGUO                  90
RUSTICA                  76
ITALIANA                 26
ESPANIOL                 21
ELEGANTE                 21
DISENIO                  18
FOLKLORICO               17
TANGO                    15
BOHEMIO                  13
CHINA                    13
AMERICANO                12
BODEGA                   11
BRITANICA                10
COLONIAL                 10
VANGUARDISTA             10
ORIENTAL                  8
MINIMALISTA               8
FUTBOL                    7
MEDITERRANEO              7
FRANCES                   6
MARINA                    6
MEXICANO                  6
ALEMANA                   5
CAMPESTRE                 5
ARABE                     4
ARMENIA                   4
ART- DECO                 4
TEATRAL                   4
POP                       4
PSICODELICO               4
CINE                      3
GRIEGA                    3
IRLANDE

In [61]:
df["categoria"].value_counts()

categoria
RESTAURANTE             1990
CAFE                     334
BAR                      247
OTROS                    116
CONFITERIA                55
PUB                       53
VINERIA                   22
SANDWICHERIA               4
DELIVERY & TAKE AWAY       2
Name: count, dtype: int64

In [ ]:
#¿Qué barrios tienen mayor presencia de propuestas saludables o alternativas?

df.groupby(["barrio","cocina"])["cocina"].count().sort_values(ascending=False).head(5)

barrio       cocina             
San Nicolas  MINUTAS                158
             INTERNACIONAL          109
Recoleta     MINUTAS                102
Palermo      GASTRONOMIA GENERAL     87
Balvanera    MINUTAS                 72
Name: cocina, dtype: int64

Podemos observar que en el barrio de San Nicolas tiene mayor presencia en minutas e internacional 

In [58]:
#¿Existen zonas con potencial de crecimiento?

df.groupby(["barrio","cocina"])["cocina"].count().sort_values(ascending=False).tail(10)

barrio            cocina         
Villa Ortuzar     SANDWICHERIA       1
Villa Pueyrredon  FAST FOOD          1
                  PORTEÑA           1
                  SANDWICHERIA       1
Villa Real        MARISCOS           1
Villa Urquiza     PANQUEQUER�?A    1
Agronomia         SANDWICHERIA       1
                  PIZZERIA           1
Villa Urquiza     SANDWICHERIA       1
                  TRADICIONAL        1
Name: cocina, dtype: int64

In [60]:
#¿Qué categorías gastronómicas tienen mayor presencia?

df.groupby(["cocina"])["cocina"].count().sort_values(ascending=False).head(5)

cocina
MINUTAS                737
INTERNACIONAL          448
GASTRONOMIA GENERAL    359
TRADICIONAL            297
PIZZERIA               176
Name: cocina, dtype: int64

#Slide1
Proyecto:
Nueva línea de productos alimenticios sostenibles

Consultora:
ComIt

#Slide2
Cliente: EcoFood


#Slide3 Datos Utilizados

Fuente:
Portal inmobiliario

Registros:
2823 propiedades

Variables:
long
lat
id 
nombre 
categoria
cocina
ambientacion
telefono
mail
horario
calle_nombre
calle_altura
calle_cruce  
direccion_completa   
barrio    
comuna                     
codigo_postal            
codigo_postal_argentino

#Slide4

1. Extracción
2. Limpieza
3. EDA
4. Visualización
5. Recomendaciones

#Slide5

Hallazgo de #¿Qué barrios tienen mayor presencia de propuestas saludables o alternativas? 
#¿Existen zonas con potencial de crecimiento?

Crear grafico con los 5 barrios con mayor y menor presencia de propuestas saludables o alternativas.
Hacer una descripcion del hallazgo y de la visualizacion.

#Slide6

¿Qué tendencias están creciendo?

¿Qué conceptos generan mayor interés?

#Slide7

¿Qué valoran más los consumidores?
¿Qué problemas aparecen con frecuencia?
¿El precio es una barrera para adoptar productos sostenibles?Incluir gráfico.

#Slide8

¿Qué productos están poco representados en el mercado?
¿Qué oportunidades existen para diferenciar una nueva marca?
¿Qué segmentos parecen tener mayor potencial?

#Slide9

- Muestra limitada.
- Datos públicos.
- No se incluyeron costos notariales.

#Slide10

Qué descubrimos
Qué significa para el negocio
Qué recomendamos hacer
